docs/syntax-reference.rst

```diff
@@ -1101,8 +1101,8 @@ Scalar functions
 |                        |                                                            | ``ksql.functions.substring.legacy.args`` to       |
 |                        |                                                            | ``true``. Though this is not recommended.         |
 +------------------------+------------------------------------------------------------+---------------------------------------------------+
-| TIMESTAMPTOSTRING      |  ``TIMESTAMPTOSTRING(ROWTIME, 'yyyy-MM-dd HH:mm:ss.SSS')`` | Converts a BIGINT millisecond timestamp value into|
-|                        |                                                            | the string representation of the timestamp in     |
+| TIMESTAMPTOSTRING      |  ``TIMESTAMPTOSTRING(ROWTIME, 'yyyy-MM-dd HH:mm:ss.SSS'    | Converts a BIGINT millisecond timestamp value into|
+|                        |    [, TIMEZONE])``                                         | the string representation of the timestamp in     |
```

Comment:
Could we include some further info and examples on how to define time zones?  So users know that they can e.g. use "UTC" or "America/Los_Angeles".

Reply:
@miguno Ready 👍

Reply:
@uurl I think @miguno meant in the documentation, not just the tests.